frontend/frames/settings.py

```diff
@@ -83,6 +93,46 @@ def __init__(self, window, base, **kwargs):
             command=self.close
         )
         self.close_button.grid(row=3, column=0)
+        self.insert_config_data(self.config)
+
+    def insert_config_data(self, config):
+        for fieldname, value in config.items():
+            if fieldname == "name":
+                self.name.set(value)
+
+    def save_config_data(self):
+        config_dir = os.path.dirname(os.path.abspath(__name__))
```

Comment:
3 first lines here and 3 first lines in `read_config` have a lot of common. Can we move them to one method?